### DISCUSSION TITLE
Moodle: using templates - key for startURL

### DISCUSSION THREAD

**Discussion by AntonT76** [SafeExamBrowser/seb-win-refactoring#436] [Q&A] [answered]
Dear community,

we would like to prepare some SEB-templates in Moodle and we have a question regarding startURL key. Our steps for creating templates are:
1. thirst we create a moodle quiz 
2. change to option "Safe Exam Browser"
3. change safe exam browser option to "Yes - configure manually" to prepare a template
4. after that we download the configuration file (*.seb) and ad it as a new template in Moodle
5. template will be enabled and can by used by teachers

We have seen that there is a startURL key which consists of the URL of the original quiz which we used for preparing the template (eg: 
<key>startURL</key><string>https://mymoodle/mod/quiz/view.php?id=1234</string>
After some tests with different quizzes it seems to be that this URL doesn't have any impact, but it causes confusion and we are not shure if there are any side effects.

Does anyone know if the startURL key has any negative effects? What happens if we delete the whole startURL key in the seb-file?

thanks, best regards
Anton

**Comment by danschlet** [comment, marked as answer]
The startURL key is populated by the Moodle SEB integration plugin according to the quiz where the template (or config file when using "YES - upload own config"). Therefore you can just leave it empty in the SEB configuration file.
